channels/api.py

```diff
@@ -603,7 +678,10 @@ def delete_post(self, post_id):
         post.delete()
         return post
 
-    @reddit_object_indexer(indexing_func=index_new_comment)
+    @reddit_object_persist(
+        index_new_comment,
+        task_helpers.sync_comment_model,
```

Comment:
In my previous comment, I meant that the imports of task helpers *in this module* should be consistent. i.e.: we import the entire search task helper module like we do for the channel task helpers (e.g.: `from search import task_helpers as search_task_helpers`) or we import the channel task helpers by name. Right now we have the search task helpers imported by name and the channel task helpers imported as a whole module. I'm ambivalent about the import rename in the test file

Reply:
No problem I'll make the same change in this one